Source/Core/DolphinQt2/GameList/GameList.cpp

```diff
@@ -90,30 +92,31 @@ void GameList::MakeListView()
 
   QHeaderView* hor_header = m_list->horizontalHeader();
 
+  hor_header->restoreState(
+      Settings::GetQSettings().value(QStringLiteral("tableheader/state")).toByteArray());
+
   connect(hor_header, &QHeaderView::sortIndicatorChanged, this, &GameList::OnHeaderViewChanged);
-  connect(hor_header, &QHeaderView::sectionResized, this, &GameList::OnHeaderViewChanged);
   connect(hor_header, &QHeaderView::sectionCountChanged, this, &GameList::OnHeaderViewChanged);
   connect(hor_header, &QHeaderView::sectionMoved, this, &GameList::OnHeaderViewChanged);
 
   if (!Settings::GetQSettings().contains(QStringLiteral("tableheader/state")))
     m_list->sortByColumn(GameListModel::COL_TITLE, Qt::AscendingOrder);
 
-  hor_header->restoreState(
-      Settings::GetQSettings().value(QStringLiteral("tableheader/state")).toByteArray());
-
   hor_header->setSectionResizeMode(GameListModel::COL_PLATFORM, QHeaderView::Fixed);
   hor_header->setSectionResizeMode(GameListModel::COL_BANNER, QHeaderView::Fixed);
-  hor_header->setSectionResizeMode(GameListModel::COL_TITLE, QHeaderView::Stretch);
+  hor_header->setSectionResizeMode(GameListModel::COL_TITLE, QHeaderView::Interactive);
   hor_header->setSectionResizeMode(GameListModel::COL_DESCRIPTION, QHeaderView::Interactive);
   hor_header->setSectionResizeMode(GameListModel::COL_MAKER, QHeaderView::Interactive);
-  hor_header->setSectionResizeMode(GameListModel::COL_ID, QHeaderView::Interactive);
+  hor_header->setSectionResizeMode(GameListModel::COL_ID, QHeaderView::Fixed);
   hor_header->setSectionResizeMode(GameListModel::COL_COUNTRY, QHeaderView::Fixed);
   hor_header->setSectionResizeMode(GameListModel::COL_SIZE, QHeaderView::Fixed);
   hor_header->setSectionResizeMode(GameListModel::COL_FILE_NAME, QHeaderView::Interactive);
 
-  m_list->setColumnWidth(GameListModel::COL_PLATFORM, Resources::GetPlatform(0).width());
-  m_list->setColumnWidth(GameListModel::COL_COUNTRY, Resources::GetCountry(0).width());
+  m_list->setColumnWidth(GameListModel::COL_BANNER, 100);
+  m_list->setColumnWidth(GameListModel::COL_PLATFORM, 40);
+  m_list->setColumnWidth(GameListModel::COL_COUNTRY, 40);
```

Comment:
For some reason, these refuse to be less than 48. Is this a known issue?

Reply:
I was able to fix this by adding `hor_header->setMinimumSectionSize(40);`

Reply:
Additionally, I think these should be changed to 38 pixels (as well as the aforementioned `setMinimumSectionSize`), since that would give them equal padding like the banners.